source/server/spongineer/logs.rst

```diff
@@ -9,6 +9,134 @@ logfiles from SpongeForge and SpongeVanilla servers including short descriptions
    :depth: 2
    :local:
 
+Configure Logging
+=================
+
+Sometimes plugins log messages that the server owner does not need or the sheer number of messages hide some more
+important information. In other cases, plugins or the server log debug messages that normally don't appear in any logs.
```

Comment:
For me; that sentence seems to be complete. I'm not sure whether it is just a missing comma.

> Who: plugins or the server
> Action: log
> What: debug messages 
> Caveat: 
> > Who: the messages 
> > Action: (normally) don't appear 
> > Where: in any logs
> > (Reason: Because they are debug messages and the console only shows info or higher)

Reply:
Aha. I had read "log" as a noun rather than a verb. Perhaps "plugins or the server *will* log" can clarify it.